src/Cache/CacheBridge.php

```
@@ -52,42 +52,44 @@ public function clear(): bool
     }
 
     /**
-     * @param array $keys
-     * @param  null  $default
-     * @return array
+     * @param iterable $keys
+     * @param mixed $default
+     * @return iterable
      */
-    public function getMultiple($keys, $default = null): array
+    public function getMultiple(iterable $keys, mixed $default = null): iterable
```

PHPMD (2.15.0)

Avoid unused parameters such as '$default'. (undefined)

(UnusedFormalParameter)